src/types/sequence.rs

```diff
@@ -239,6 +239,16 @@ impl PySequence {
 
     /// Returns a fresh list based on the Sequence.
     #[inline]
+    pub fn to_list(&self) -> PyResult<&PyList> {
+        unsafe {
```

Comment:
Personally, I would prefer if one implementation, e.g. the deprecated one, would forward to the other to avoid the code duplication. I do understand that this is probably not worth it for one-liners, on the other hand these are `unsafe` one-liners.

Reply:
Agreed, I meant to do this and then forgot while actually doing the typing! Have force-pushed to correct.